### PR TITLE
Refactor Terms of Use section to use EBSI examples.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to partially address issue #1437 by updating the examples in the `termsOfUse` section to use examples from the work that EBSI has deployed.

The PR will need further refinement from @alenhorvat for some of the values in the examples. Please suggest changes to the PR with the appropriate values for the example (and any additional language you feel might be useful).

This PR is a replacement form PR #1498.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1533.html" title="Last updated on Aug 3, 2024, 8:41 PM UTC (2bfd656)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1533/ccbe460...2bfd656.html" title="Last updated on Aug 3, 2024, 8:41 PM UTC (2bfd656)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3532,97 +3532,30 @@ <h3>Terms of Use</h3>
...
   },
   "credentialSchema": {
-    "id": "https://api-test.ebsi.eu/trusted-schemas-registry/v2/schemas/z3MgUFUkb722uq4x3dv5yAJmnNmzDFeK5UC8x83QoeLJM",
+    "id": "https://code.europa.eu/ebsi/json-schema/-/raw/db67c771b96715c23ddf384abcfef33d6174d494/schemas/vcdm2.0/trust-model/schema.json",
     "type": "JsonSchema"
   },
   "termsOfUse": {
-    "id": "https://api-test.ebsi.eu/trusted-issuers-registry/v4/issuers/did:ebsi:zz7XsC9ixAXuZecoD9sZEM1/attributes/7201d95fef05f72667f5454c2192da2aa30d9e052eeddea7651b47718d6f31b0",
-    "type": "IssuanceCertificate"
+    "type": "TrustFrameworkPolicy",
```
Comment (David-Chadwick) [inline index.html:3539]:
```suggestion
    "type": "AccreditationPolicy",
```

Comment (David-Chadwick) [reply index.html:3539]: Note that the EBSI documentation is still evolving and is internally inconsistent. For example 
https://hub.ebsi.eu/vc-framework/trust-model/issuer-trust-model-v4
says "the issued domain VC must contain a termsOfUse property with AttestationPolicy type"
whilst the schema does not contain this value.

File: index.html [hunk 2]
```diff
@@ -3532,97 +3532,30 @@ <h3>Terms of Use</h3>
...
+    "trustFramework": "EBSI_TRUST_FRAMEWORK_URI_GOES_HERE",
```
Comment (David-Chadwick) [inline index.html:3540]:
```suggestion
    "parentAccreditation": "URL to access the parent entity in the trust chain supporting the attestation",
```

File: index.html [hunk 3]
```diff
@@ -3532,97 +3532,30 @@ <h3>Terms of Use</h3>
...
+    "policyId": "EBSI_POLICY_ID_URI_GOES_HERE"
```
Comment (David-Chadwick) [inline index.html:3541]:
```suggestion
    "rootAuthorisation": "URL to access the root entity in the trust chain supporting the attestation"
```

File: index.html [hunk 4]
```diff
@@ -3532,97 +3532,30 @@ <h3>Terms of Use</h3>
...
   }
 }
         </pre>
 
         <p>
-In the example above, the [=issuer=] is asserting that as a European
```
Comment (David-Chadwick) [inline index.html:3546]: Please do not remove lines 3546 to 3552 as this description is still approximately correct

**Review by David-Chadwick — CHANGES_REQUESTED** [review]
This modification does not help as it is changing the example. The existing example (prior to this PR) was taken from the previous EBSI documentation (which has been modified this year) and shows the ToU that an issuer inserts into an end user's VC. So I suggest it will be easier for EBSI to update the existing example to one using their updated documentation.

**Comment by msporny** [comment]
> This modification does not help as it is changing the example.

This PR is a placeholder until EBSI can provide the latest examples directly:

https://lists.w3.org/Archives/Public/public-vc-wg/2024Jul/0013.html

File: index.html (hunk 1 above)
Comment (yhuard) [reply index.html:3539]: Hi @David-Chadwick. The schema https://code.europa.eu/ebsi/json-schema/-/blob/main/schemas/vcdm2.0/trust-model/schema.json?ref_type=heads is used for accreditations only (Root TAO and TAO)
The `AttestationPolicy` type is defined in the type extension https://code.europa.eu/ebsi/json-schema/-/tree/main/schemas/vcdm2.0/type-extensions/termsOfUse/AttestationPolicy?ref_type=heads What's missing in EBSI's documentation is how to resolve type extensions.

**Review by decentralgabe — APPROVED** [review]

**Comment by msporny** [comment]
@David-Chadwick I updated the example to be what EBSI provided, which is what we needed to address issue #1437.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made (except for those not aligned with what EBSI provided), no objections, merging.

**Pull request merged**